### DISCUSSION TITLE
Getting "too low - terrain" warnings when landing

### DISCUSSION THREAD

**Discussion by flyingpaul1980** [flybywiresim/aircraft#2727] [Q&A]
Hi, 

even though my a320 is configured for landing (all green - no blue), the ATC gave me landing clearance, and i'm on the glide slope, it gives me "too low - terrain" warnings on my final approach.

Any Idea what might be causing this? 

Greetings, Paul

**Comment by MapleNAV-TCASFAULT** [comment]
Did you activate APPR phase? as the aircraft may be thinking you're not going for an approach yet. But to turn off the sounds hit the TERR and SYS on the GPWS panel 
![InkedOverhead_panel_of_an_Airbus_A320_during_cruise_LI](https://user-images.githubusercontent.com/72575144/103156820-d92b2e00-477a-11eb-8407-ff6aee93d105.jpg)

**Reply by flyingpaul1980** [reply]
Yes approach activated in both mcdu and autopilot, autopilot deactivated below 1000ft above or so and then it began. Thx for the overhead panel advice!

**Comment by robinottawa** [comment]
Real320simpilot laughed when he heard that and said something about a new bug. I noticed it too and just ignore it.

**Comment by ghost** [comment]
Are you using Flaps 3 for landing? Flaps 3 landings are not implemented yet.

**Comment by BozoBBozo** [comment]
I had this happen today, flying into OMDB Dubai, being perfectly on the glideslope and localizer in Approach mode. However, that GS puts you just barely above some ~10 story buildings on approach, such that newer tires might have scraped. The warning was repeated from the first tall building all the way up until landing - it continued sounding even after I was over flat ground on the airport and even the runway threshold, until touching down.
